R/utils.R

```diff
@@ -51,9 +51,33 @@ is_logical_vector_modif <- function(x, min_length = 1) {
 }
 # nocov end
 
-#' Currently supported cell value alignments
-#' @export
+#' @title Alignment utils
+#'
+#' @description Currently supported cell value alignments. These values 
+#'   may be used to set single cell alignment (`align` in [rcell()]) or row-wise 
+#'   alignment (`.aligns` in [in_rows()]).
+#' 
 #' @return a vector of alignments currently supported.
+#' 
+#' @examples 
+#' # Right alignment of single cell
```

Comment:
examples should include call to the function documented (in this case `rtables_aligns`)